Source/Dafny/Resolver.cs

```diff
@@ -1651,7 +1652,7 @@ public class ModuleBindings {
                : "");
     }
 
-    [Pure]
+    [System.Diagnostics.Contracts.Pure]
```

Comment:
I don't know if the one from JetBrains means the same thing. The reason for this `[Pure]` is so that the method can be used in `CodeContracts` expressions (which are not allowed to have side effects). Since we're no longer running the `CodeContracts` tools on the Dafny implementation, I just removed this annotation instead.